presto-main/src/main/java/com/facebook/presto/sql/planner/ExpressionInterpreter.java

```diff
@@ -1044,6 +1046,12 @@ protected Object visitLikePredicate(LikePredicate node, Object context)
                     optimizedEscape);
         }
 
+        private boolean typeCanSerialize(Type type)
+        {
+            return isSupportedLiteralType(type) || type.getJavaType().isAssignableFrom(Block.class)
```

Comment:
Why `Block` is considered as `serializable`? 

Also, my understanding is it's about whether the value can be encoded in plan (and distribute to worker nodes) right? So maybe call it `typeCanBeEncodedInPlan`?